pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -2790,9 +2790,16 @@ func (c *Cloud) EnsureLoadBalancer(clusterName string, apiService *v1.Service, n
 	return status, nil
 }
 
+// GetLoadBalancerName returns the name of the load balancer based on the service UID.
+func (c *Cloud) GetLoadBalancerName(service *v1.Service) string {
+	ret := string(service.UID)
```

Comment:
.. What am I missing. Shouldn't all these `GetLoadBalancerName()` implementations call `DeprecatedGetLoadBalancerName()` right now?  That way we have a no-op until (and if) a cloud provider chooses to do something else.

Afaics (looking at the openstack provider), the current change adds a new/different implementation of `openstack.GetLoadBalancerName()`, and then modifies all the places in openstack code to call `DeprecatedGetLoadBalancerName()`, which seems wrong.